docs/source/sklearn_integrations.mdx

```diff
@@ -0,0 +1,80 @@
+# Scikit-Learn
+
+To run the scikit-learn examples make sure you have installed the following libraries:
+
+```bash
+pip install -U scikit-learn
+```
+
+The metrics in `evaluate` can be easily integrated with an Scikit-Learn estimator or [pipeline](https://scikit-learn.org/stable/modules/generated/sklearn.pipeline.Pipeline.html#sklearn.pipeline.Pipeline).  
+
+However, these metrics require that we generate the predictions from the model. The predictions and labels from the estimators can be passed to `evaluate` mertics to compute the required values.
+
+```python
+import numpy as np
+import evaluate
+from sklearn.compose import ColumnTransformer
+from sklearn.datasets import fetch_openml
+from sklearn.pipeline import Pipeline
+from sklearn.impute import SimpleImputer
+from sklearn.preprocessing import StandardScaler, OneHotEncoder
+from sklearn.linear_model import LogisticRegression
+from sklearn.model_selection import train_test_split
+
+np.random.seed(0)
+
+# Load data from https://www.openml.org/d/40945
+
+X, y = fetch_openml("titanic", version=1, as_frame=True, return_X_y=True)
+
+# Alternatively X and y can be obtained directly from the frame attribute:
```

Comment:
I would break the code block where you have some natural language comment so these comments are displayed as normal text. Reading a long code block can be a bit intimidating but if you break it is easier to read.